Description: Use ntpsec as the user
 This is the user used in the ntpsec packaging, to stay out of the namespace
 of the ntp package.
Forwarded: not-needed
Origin: vendor
Author: Richard Laager <rlaager@debian.org>
Last-Update: 2020-11-16
--- a/docs/includes/ntpkeygen-body.adoc
+++ b/docs/includes/ntpkeygen-body.adoc
@@ -32,7 +32,7 @@
 The simplest way to run the +ntpkeygen+ program is logged in directly as
 root. The recommended procedure is to change to the keys directory,
 usually +/etc/ntp/+, then run the program.  Then chown the output
-file to ntp:ntp.  It should be mode 400.
+file to ntpsec:ntpsec.  It should be mode 400.
 
 WARNING: +ntpkeygen+ uses the system randomness source.  On a POSIX
 system, this is usually /dev/urandom.  Immediately after a reboot, on any
--- a/docs/NTS-QuickStart.adoc
+++ b/docs/NTS-QuickStart.adoc
@@ -122,7 +122,7 @@
 Note that `ntpd` must be able to read both files and you want to
 make sure that the bad guys can't read your private key.  It may
 be simpler to copy those files over to `/etc/ntp/` and adjust
-their owner and mode so `ntpd` running as user `ntp` can read them.
+their owner and mode so `ntpd` running as user `ntpsec` can read them.
 
 You may need to tell your system where to store the keys used
 to encrypt cookies.  The default is `/var/lib/ntp/nts-keys`.
